### DISCUSSION TITLE
[Dracula PRO - Windows Terminal] PowerShell parameters are too dark

### DISCUSSION THREAD

**Discussion by pyxelr** [dracula/dracula-theme#715] [Ideas]
I have observed that the colour of parameters passed in the PowerShell profile is too dark for Dracula PRO (it is not the case of the free Dracula).

Let me present a couple of examples with the `-Path` parameter. In all of them, I am applying 80% background opacity of Windows Terminal, where the situation gets slightly better if I would not apply it. Besides, I am using the starship prompt, which does not affect the colouring.

- Example 1 (focused window on a darker background):
![1](https://user-images.githubusercontent.com/30161753/166320454-93a6f076-c131-409b-803e-dee3e3f569f5.png)

- Example 2 (unfocused window on a darker background):
![2](https://user-images.githubusercontent.com/30161753/166320471-0fb01198-ec20-4f5f-a04d-454a0f7f48d2.png)

- Example 3 (focused window on a brighter background):
![3](https://user-images.githubusercontent.com/30161753/166320482-fd8e8c73-d00f-44b4-b011-5df63f071427.png)

---

To improve the readability, I propose replacing `"brightBlack": "#504C67"` with `"brightBlack": "#8F89B4"`.
Here is the effect:

- Example 1 (focused window on a darker background):
![4](https://user-images.githubusercontent.com/30161753/166320508-b7ee5d95-5307-4879-8f0b-bd586b5cc043.png)

- Example 2 (unfocused window on a darker background):
![5](https://user-images.githubusercontent.com/30161753/166320518-730468e7-1641-45f5-8a94-3d04093e4db3.png)

**Comment by luxonauta** [comment]
Maybe @waf can join the discussion? 🤔

**Reply by waf** [reply]
I'm not sure what setting dracula pro is using, but I've found these settings to work well for PowerShell:
 https://github.com/waf/config/blob/master/Microsoft.PowerShell_profile.ps1#L40L56

I use it with these Windows Terminal settings:
 https://github.com/dracula/powershell/blob/master/theme/Windows%2010%20Terminal%20Dracula%20Theme.json

**Comment by leandrofelix-dev** [comment]
Interessante
